worlds/lufia2ac/__init__.py

```diff
@@ -0,0 +1,351 @@
+            self.starting_party.value = 0x00000001
+
+    def create_regions(self) -> None:
+        menu = Region("Menu", RegionType.Generic, "Menu", self.player, self.multiworld)
+        menu.exits.append(Entrance(self.player, "AncientDungeonEntrance", menu))
+        self.multiworld.regions.append(menu)
+
+        ancient_dungeon = Region("AncientDungeon", RegionType.Generic, "Ancient Dungeon", self.player, self.multiworld)
+        ancient_dungeon.exits.append(Entrance(self.player, "FinalFloorEntrance", menu))
+        item_count: int = self.blue_chest_count
+        if self.capsule_monsters_available == CapsuleMonstersAvailable.option_shuffled_into_multiworld:
+            item_count += len(self.item_name_groups["Capsule monsters"])
+        if self.party_members_available == PartyMembersAvailable.option_shuffled_into_multiworld:
+            item_count += len(self.item_name_groups["Party members"])
+        for location in l2ac_location_table[:item_count]:
+            location_id: int = self.location_name_to_id[location.name]
+            ancient_dungeon.locations.append(L2ACLocation(self.player, location.name, location_id, ancient_dungeon))
+        prog_chest_access = L2ACItem("Progressive chest access", ItemClassification.progression, None, self.player)
+        for i in range(CHESTS_PER_SPHERE, item_count, CHESTS_PER_SPHERE):
+            chest_access = \
+                L2ACLocation(self.player, f"Chest access {i + 1}-{i + CHESTS_PER_SPHERE}", None, ancient_dungeon)
+            chest_access.place_locked_item(prog_chest_access)
+            ancient_dungeon.locations.append(chest_access)
+        treasures = L2ACLocation(self.player, "Iris Treasures", None, ancient_dungeon)
+        treasures.place_locked_item(L2ACItem("Treasures collected", ItemClassification.progression, None, self.player))
+        ancient_dungeon.locations.append(treasures)
+        self.multiworld.regions.append(ancient_dungeon)
+
+        final_floor = Region("FinalFloor", RegionType.Generic, "Ancient Cave Final Floor", self.player, self.multiworld)
+        boss = L2ACLocation(self.player, "Boss", None, final_floor)
+        boss.place_locked_item(L2ACItem("Boss victory", ItemClassification.progression, None, self.player))
+        final_floor.locations.append(boss)
+        self.multiworld.regions.append(final_floor)
+
+        self.multiworld.get_entrance("AncientDungeonEntrance", self.player) \
+            .connect(self.multiworld.get_region("AncientDungeon", self.player))
+        self.multiworld.get_entrance("FinalFloorEntrance", self.player) \
+            .connect(self.multiworld.get_region("FinalFloor", self.player))
+
+    def create_items(self) -> None:
+        item_pool: List[str] = \
+            self.multiworld.random.choices(sorted(self.item_name_groups["Blue chest items"]), k=self.blue_chest_count)
+        if self.capsule_monsters_available == CapsuleMonstersAvailable.option_shuffled_into_multiworld:
```

Comment:
you seem to look for these exact comparators multiple times, so it's probably best to make a property for this to improve speed and reduce typing.